src/script/components/deviceCard.ts

```diff
@@ -1,111 +0,0 @@
-/*
- * Wire
- * Copyright (C) 2018 Wire Swiss GmbH
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see http://www.gnu.org/licenses/.
- *
- */
-
-import ko from 'knockout';
-import {ClientClassification} from '@wireapp/api-client/src/client';
-
-import type {ClientEntity} from '../client/ClientEntity';
-
-interface DeviceCardParams {
-  click?: (device: ClientEntity) => void;
-  current?: boolean;
-  detailed?: boolean;
```

Comment:
What happened to those params? Are they not used anymore?

Reply:
nope, they are not used at all so I removed them.